src/quoter.js

```diff
@@ -83,28 +83,34 @@ Quoter.prototype = {
   },
 
   answerLiquidity (req) {
-    const curve = this.findCurve(req.destinationAccount)
-    return Promise.resolve({
-      liquidityCurve: curve.buf,
-      appliesToPrefix: curve.prefix,
-      sourceHoldDuration: 15000,
-      expiresAt: new Date(Date.now() + 3600 * 1000)
+    return Promise.resolve().then(() => {
```

Comment:
What is the reason for this change?

Reply:
Because https://github.com/interledgerjs/amundsen/blob/master/src/request-handler.js#L34 requires this function to return a promise.

Reply:
That's not what I mean.

```
return Promise.resolve().then(() => {
    return Promise.resolve(foo)
})
```

is redundant. You can just ```return Promise.resolve(foo)``` for the same effect.

Reply:
corrected.